lib/yard-sorbet/sig_handler.rb

```diff
@@ -117,7 +123,7 @@ def process
           if p.type == :assoc
             param_name = p.children.first.source[0...-1]
             types = YARDSorbet::SigToYARD.convert(p.children.last)
-            parsed.params[param_name] = types
+            parsed.params[param_name] = ParsedParam.new({ types: types, docstring: p.docstring })
```

Comment:
Is this style consistent with the rest of the code, passing a hash?

Reply:
the code doesn't pass anything, but it's inconsistent with the sort of default style modeled in the Sorbet T::Struct docs, so I'm going to change it!